<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>admin-client-win  该客户端请不要滥用,请尽量只用于更正数据</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>810</width>
      <height>603</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_info">
     <attribute name="title">
      <string>个人信息</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>780</width>
        <height>250</height>
       </rect>
      </property>
      <property name="title">
       <string>个人资料展示</string>
      </property>
      <widget class="QLabel" name="label_profile_photo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>200</width>
         <height>210</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>头像</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_info">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>510</width>
         <height>210</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>姓名：老母鸡

账号：12345</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>780</width>
        <height>280</height>
       </rect>
      </property>
      <property name="title">
       <string>个人资料修改</string>
      </property>
      <widget class="QLabel" name="label_name">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>500</width>
         <height>40</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>请输入需要修改的姓名</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_name">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>30</y>
         <width>160</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>500</width>
         <height>40</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>输入需要修改的密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_password">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_password">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>80</y>
         <width>160</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_profile_photo">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>500</width>
         <height>40</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>点击修改会自动把图片上传到服务器并修改头像</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_photo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>头像：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_profile_photo">
       <property name="geometry">
        <rect>
         <x>685</x>
         <y>130</y>
         <width>75</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_logout">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>720</width>
         <height>70</height>
        </rect>
       </property>
       <property name="text">
        <string>退出(取消自动登录)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_open_file">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>130</y>
         <width>75</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>选图</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_book">
     <attribute name="title">
      <string>书籍管理</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_book">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>820</width>
        <height>550</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>90</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>书籍编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>标题</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>作者/出版社</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>书籍类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>录入的管理员</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>是否可用</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>不可用原因</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>书籍录入时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>条形码</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_borrow">
     <attribute name="title">
      <string>借阅管理</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_borrow">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>810</width>
        <height>560</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>110</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>借阅编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>书籍编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>同意的管理员</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>借书的用户</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开始借阅的时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>归还的时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>书籍状态</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_user">
     <attribute name="title">
      <string>用户管理</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_user">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>810</width>
        <height>560</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>130</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>用户账号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>密码</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>部门</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>帐号状态</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>头像文件名</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
